validator/validator.go

```diff
@@ -167,3 +167,9 @@ func (v *Validator) GetIBFT(r beacon.RoleType) (ibft.IBFT, bool) {
 	i, exist := v.ibfts[r]
 	return i, exist
 }
+
+// GetQueue returns msgQueue
+func (v *Validator) GetQueue() *msgqueue.MessageQueue {
```

Comment:
@nivBlox why do we need this?

Reply:
@amir-blox related to metric (moving to separate pr)

Reply:
@nivBlox doesn't seem to be moved